docs/live-response.rst

```diff
@@ -0,0 +1,84 @@
+Live Response
+==============
+
+With the Carbon Black Cloud SDK, we can use the Live Response functionality to upload, download, and remove files,
+retrieve and remove registry entries, dump contents of physical memory, execute and terminate processes.
+Before any commands can be sent to an endpoint, we must first establish a “session” with a device.
+
+Establish A Session With A Device
+---------------------------------
+Using a query of the ``Device`` object, we can choose a device to connect to.
+
+::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> api = CBCloudAPI(profile='sample')
+    >>> from cbc_sdk.platform import Device
+    >>> device = api.select(Device).first()
+    >>> lr_session = device.lr_session()
+
+File Commands
+-------------
+
+Once a session is established we are going to create a directory and upload a file in that newly created directory.
+List directory command is returning the content of the directory including the uploaded file.
+
+::
+
+    >>> lr_session.create_directory('C:\\\\demo\\\\')
+    >>> lr_session.put_file(open("demo.txt", "r"), 'C:\\\\demo\\\\demo.txt')
+    >>> directories = lr_session.list_directory('C:\\\\demo\\\\')
+    >>> for directory in directories:
+    ...   print(f"{directory['attributes'][0]} {directory['filename']}")
+    ...
+    DIRECTORY .
+    DIRECTORY ..
+    ARCHIVE demo.txt
+
+Note that the creation of the directory will fail if the directory already exists.
+
+Now we will get the contents of the file and then delete the file and the directory.
```

Comment:
Consider changing to:
"Next, get the contents of the file and then delete the file and the directory."